helminator.py

```diff
@@ -205,6 +205,7 @@ def get_chart_updates():
                 chart_updates.append(repo_chart)
                 logging.info(f"found update for chart '{repo_chart['name']}': "
                              f"{ansible_chart_version} -> {latest_version}")
+                break
```

Comment:
what happen if there are more newer version?
I also think there must be a better way ;-)

Reply:
True - I opened an issue